src/VisualStudio/Xaml/Impl/Implementation/LanguageServer/Handler/Completion/CompletionHandler.cs

```diff
@@ -89,6 +89,16 @@ private static CompletionItem CreateCompletionItem(XamlCompletionItem xamlComple
                 };
             }
 
+            if (xamlCompletion.EventDescription.HasValue)
+            {
+                item.Command = new Command()
+                {
+                    CommandIdentifier = StringConstants.CreateEventHandlerCommand,
+                    Arguments = new object[] { textDocument, xamlCompletion.EventDescription },
+                    Title = Resources.CreateEventHandler
```

Comment:
Is the title shown anywhere for completion commands?  If it's not we may not need to localize it.

Reply:
You have a good point. I am not aware the title is shown anywhere so far. I have removed the string from the resources.

Reply:
Small improvement, make it a const string in this class.

---
In reply to: [587847403](https://github.com/dotnet/roslyn/pull/51670#discussion_r587847403) [](ancestors = 587847403)